lib/init.g

```diff
@@ -1019,8 +1019,11 @@ end );
 if IsHPCGAP and THREAD_UI() then
   ReadLib("hpc/consoleui.g");
   MULTI_SESSION();
-else
+elif not IsBound( IsLIBGAP ) or IsLIBGAP = false then
+  BIND_CONSTANT( "IsLIBGAP", false );
```

Comment:
Test fail, partially because of your .tst file... e.g. in HPC-GAP, we will *not* define `IsLIBGAP`. Better is to do this *before* line 1019:
```gap
  if not IsBound(IsLIBGAP) then
    BIND_CONSTANT( "IsLIBGAP", false );
  fi;
```
and then change this check here back to `elif not IsLIBGAP`, and drop the `else` part.